src/accepted-cog-list.json

```
@@ -663,6 +671,12 @@
 
   "69149",
 
+  "77047",
+
+  "80261",
+
+  "85330",
```

```suggestion
  "85163",
``` 85330 est le code postal

---

Erreur dans le formulaire, le COG est bien 85163